### PR TITLE
chore(shell-api): Update deprecation messages for collection crud commands MONGOSH-522

### PR DESCRIPTION
**Pull request by gribnoysup**
https://jira.mongodb.org/browse/MONGOSH-522

### CODE REVIEW

File: packages/shell-api/src/collection.ts [hunk 1]
```diff
@@ -782,7 +784,7 @@ export default class Collection extends ShellApiClass {
   @serverVersions([ServerVersions.earliest, '3.2.0'])
   async update(filter: Document, update: Document, options: UpdateOptions & { multi?: boolean } = {}): Promise<UpdateResult | Document> {
     printDeprecationWarning(
-      'Collection.update() is deprecated. Use updateOne, updateMany or bulkWrite.',
+      'Collection.update() is deprecated. Use updateOne, updateMany, findAndModify, or bulkWrite.',
```
Comment (addaleax) [inline packages/shell-api/src/collection.ts:787]: Fun fact: `findAndModify` is deprecated as a driver function: https://github.com/mongodb/node-mongodb-native/pull/2740#issuecomment-782399075

I think we want to keep `findAndModify` a shell method, but maybe we don't want to recommend using it anymore?

Comment (gribnoysup) [reply packages/shell-api/src/collection.ts:787]: Ah, thanks for pointing this out! Yeah, then it makes sense to me to not include it here. I'll ping Jeff in the ticket about that before removing it from the PR

File: packages/shell-api/src/collection.ts [hunk 2]
```diff
@@ -737,7 +737,9 @@ export default class Collection extends ShellApiClass {
   @returnsPromise
   @deprecated
   save(): Promise<void> {
-    throw new MongoshInvalidInputError('Collection.save() is deprecated. Use insertOne, insertMany, updateOne or updateMany.');
+    throw new MongoshInvalidInputError(
+      'Collection.save() is deprecated. Use insertOne, insertMany, updateOne, updateMany, or findAndModify.'
```
Comment (gribnoysup) [inline packages/shell-api/src/collection.ts:741]:
```suggestion
      'Collection.save() is deprecated. Use insertOne, insertMany, updateOne, or updateMany.'
```

File: packages/shell-api/src/collection.ts (hunk 1 above)
Comment (gribnoysup) [inline packages/shell-api/src/collection.ts:787]:
```suggestion
      'Collection.update() is deprecated. Use updateOne, updateMany, or bulkWrite.',
```

Comment (gribnoysup) [reply packages/shell-api/src/collection.ts]: Removed in 6c0dce6

**Pull request merged**